docs/installing.rst

```diff
@@ -55,8 +74,69 @@ Install behind a proxy::
   python get-pip.py --proxy="[user:passwd@]proxy.server:port"
 
 
-Upgrade pip
------------
+Installing with Linux Package Managers
+--------------------------------------
+
+Fedora
+~~~~~~
+
+To install the Fedora supported version of pip:
+
+* < Fedora 23:
+
+ * Python 2: ``sudo yum install python-pip``
+ * Python 3: ``sudo yum install python3-pip``
+
+* >= Fedora 23:
+
+ * Python 2: ``sudo dnf install python-pip``
+ * Python 3: ``sudo dnf install python3-pip``
+
+To get newer versions of pip (and also setuptools and wheel), you can enable the
+"unofficial" `PyPA Copr Repo <https://copr.fedoraproject.org/coprs/pypa/pypa/>`_
+using `these instructions
+<https://fedorahosted.org/copr/wiki/HowToEnableRepo>`__, and run the same
+commands as above.
+
+
+CentOS/RHEL
+~~~~~~~~~~~
+
+CentOS and RHEL don't offer pip in their core repositories.
```

Comment:
Yeah, the upstream -> SCL pipeline is still too slow (and manual). We're working on fixing that, but it took a long time to find a good upstream home for ongoing SCL maintenance (we were originally going to do something in Fedora, but the Fedora community weren't keen, so Honza has been working on setting it up as a CentOS SIG instead).

This kind of situations is one of the major downsides of the "go faster" folks generally refusing to pay for open source software, but also not being willing to help out with distro software development pipeline improvements either :P